Image refresh part 7 (#7752)
fishtown-analytics/docs.getdbt.com: website/docs/guides/redshift-qs.md

## Connect dbt to Redshift 
1. Create a new project in [<Constant name="cloud" />](/docs/cloud/about-cloud/access-regions-ip-addresses). Navigate to **Account settings** (by clicking on your account name in the left side menu), and click **+ New Project**.
2. Enter a project name and click **Continue**.

> 3. In the **Configure your development environment** section, click the **Connection** dropdown menu and select **Add new connection**.

Reviewer: ```suggestion
3. In the **Configure your development environment** section, click the **Connection** dropdown menu and select **Add new connection**.  This directs you to the connection configuration settings. 
```

>   This directs you to the **Add new connection** page. 

Reviewer: I would keep this in the line above
```suggestion
```

4. In the **Type** section, select **Redshift**.
5. Enter your Redshift settings. Reference your credentials you saved from the CloudFormation template.
    - **Hostname** &mdash; Your entire hostname.
    - **Port** &mdash; `5439`

>     - **Database** &mdash; `dbtworkshop`.

Author: Database is not in the UI anymore -- delete?

>     - **Database** &mdash; `dbtworkshop`.

Reviewer: It's hidden by default. We should call that out here
```suggestion
    - **Database** (under **Optional settings**) &mdash; `dbtworkshop`
```

8. Select your project that uses the Redshift connection. 
9. Click the **configure your development environment and add a connection** link. This directs you to a page where you can enter your personal development credentials.
10. Set your development credentials. These credentials will be used by <Constant name="cloud" /> to connect to Redshift. Those credentials (as provided in your CloudFormation output) will be:
    - **Username** &mdash; `dbtadmin`
    - **Password** &mdash; This is the autogenerated password that you used earlier in the guide
    - **Schema** &mdash; <Constant name="cloud" /> automatically generates a schema name for you. By convention, this is `dbt_<first-initial><last-name>`. This is the schema connected directly to your development environment, and it's where your models will be built when running dbt within the <Constant name="cloud_ide" />.

    <Lightbox src="/img/redshift_tutorial/images/dbt_cloud_redshift_development_credentials.png" title="dbt - Redshift Development Credentials" />

11. Click **Test connection**. This verifies that <Constant name="cloud" /> can access your Redshift cluster.

> 12. If the test succeeded, you can set up a repository. If it failed, you might need to check your Redshift settings and credentials.

Reviewer: ```suggestion
12. If the test succeeded, click **Save** to complete the configuration. If it failed, you might need to check your Redshift settings and credentials.
```
